java/src/jmri/jmrix/dcc4pc/Dcc4PcSensorManager.java

```diff
@@ -362,9 +351,9 @@ public static String decodeInputState(int state) {
     public final static int CONSIST_ADDRESS = 0x08;
 
     /**
-     * Determining if the railcommdata is duplicated. If it is then this
+     * Determining if the railcomm data is duplicated. If it is then this
      * instructs the rc input to move things about.
-     * @param value Railcommand data
+     * @param value Railcomm and data
```

Comment:
I think this "Railcomand" was the long name for "Railcom", so this comment could be "Railcom data"

Reply:
Thanks, this makes more sense.